include/chia/copy.h

```diff
@@ -17,10 +17,17 @@
 #include <cstdio>
 #include <cstdint>
 
+#include <filesystem>
```

Comment:
that has created many problems before, gotta souround all of it with `#ifdef __linux__`

Reply:
thanks, no real programming experience so forgive my terrible coding...